### DISCUSSION TITLE
[E002] Can't find factory for 'transformer' in Spanish (es)

### DISCUSSION THREAD

**Discussion by JessitaMS** [explosion/spaCy#9926] [Help: Installation] [answered]
I have a problem using es_dep_news_trf, give me the following error: KeyError: "[E002] Can't find factory for 'transformer'. This usually happens when spaCy calls `nlp.create_pipe` with a component name that's not built in - for example, when constructing the pipeline from a model's meta.json. If you're using a custom component, you can write to `Language.factories['transformer']` or remove it from the model meta and add it via `nlp.add_pipe` instead.".

I read in this space if I install this: pip install spacy[transformers] maybe the problem fixed, but isn`t. So, I have a doubt what should I do?

**Comment by polm** [comment, marked as answer]
Transformer models require a separate package, `spacy-transformers`, which is why you need to install `spacy[transformers]`. If you have that installed it should work, or at least you shouldn't get that error. Can you provide the output of `spacy info` and `pip freeze`? It's possible your environment got in a weird state. I would recommend reinstalling from scratch in a clean environment, using `pip install spacy[transformers]` to start with.
